appium/options/base_options_descriptor.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, TypeVar, Generic, Type, Callable, Optional
+
+from appium.options.common.supports_capabilities import SupportsCapabilities
+
+R = TypeVar('R')
+V = TypeVar('V')
+C = TypeVar('C', bound='SupportsCapabilities')
+
+class OptionsDescriptor(Generic[R, V]):
+    """Generic Descriptor class which calls get_capability and set_capability:
+    without transforming the values
+
+    If transformation method is passed, get_capability and set_capability:
+    method will be called with transformed value.
+    """
+    def __init__(self, name: str,
+                 tget: Optional[Callable[[Any], R]] = None, 
+                 tset: Optional[Callable[[V], Any]] = None) -> None:
+        self.name = name
+        self.tget = tget
+        self.tset = tset
+
+    def __get__(self, obj: C, cls: Type[C]) -> R:
+        value = obj.get_capability(self.name)
+        if not self.tget is None:
```

Comment:
prefer `is not None` syntax: https://stackoverflow.com/questions/2710940/python-if-x-is-not-none-or-if-not-x-is-none

Reply:
Fixed. Changed to `is not None`